hedera-node/hedera-file-service-impl/src/main/java/com/hedera/node/app/service/file/impl/schemas/GenesisSchema.java

```diff
@@ -124,14 +124,14 @@ private void createGenesisFeeSchedule(
             final var feeScheduleJsonBytes = requireNonNull(in).readAllBytes();
             final var feeSchedule = parseFeeSchedules(feeScheduleJsonBytes);
             final var fileNum = filesConfig.feeSchedules();
-            final var fileId = FileID.newBuilder().fileNum(fileNum).build();
+            final var fileId = FileID.newBuilder().fileNum(fileNum).build(); // default to shard=0, realm=0
```

Comment:
We should file an issue to come back to these places and use the shard/realm defined in config for them

Reply:
We need to create this issue for all services.

Reply:
I went ahead and implemented it for FileID. It was easy, quicker than filing an issue.